receiver/riakreceiver/metadata.yaml

```diff
@@ -0,0 +1,67 @@
+name: riakreceiver
+
+attributes:
+  riak.node.name:
+    description: The name of the Riak node.
+  operation:
+    value: state
+    description: The operation type.
+    enum:
+      - put
+      - get
+      - read
+      - write
+      - delete
+
+
+metrics:
+  riak.node.operation.count:
+    description: The number of operations performed by the node.
+    unit: "{operation}"
+    sum:
+      monotonic: true
+      aggregation: cumulative
+      value_type: int
+    enabled: true
+    attributes: [operation]
+  riak.node.operation.time.mean:
+    description: The mean time between request and response for operations performed by the node.
+    unit: "ms"
+    gauge:
+      value_type: int
+    enabled: true
+    attributes: [operation]
+  riak.node.read_repair.count:
+    description: The number of read repairs performed by the node over the last minute.
```

Comment:
Based on the [documentation](https://docs.riak.com/riak/kv/2.2.3/using/cluster-operations/inspecting-node.1.html#total-stats), it looks like there is a monotonic counter for essentially the same metric, which would be preferred. The benefit of using a monotonic counter is that there is no ambiguity about how many read repairs occurred over longer intervals. 

For example, if you collect every 2 minutes, the non-monotonic counter won't tell you what happened during the first half of the interval. If you collect every 30 seconds, there is overlap, which is difficult to deal with. Even if you collect every minute, there are realistically small gaps and/or overlaps. Since there is a non-monotonic counter, let's switch to that.